go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/notaryproject/notation
 go 1.20
 
 require (
-	github.com/notaryproject/notation-core-go v1.0.0-rc.3
-	github.com/notaryproject/notation-go v1.0.0-rc.5
+	github.com/notaryproject/notation-core-go v1.0.0-rc.4
+	github.com/notaryproject/notation-go v1.0.0-rc.6
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc.3
+	github.com/opencontainers/image-spec v1.1.0-rc3
 	github.com/oras-project/oras-credentials-go v0.1.1-0.20230517070118-6b1be42cc39f
```

Comment:
There is new version of oras-credentials-go  `v0.1.1`, see https://github.com/oras-project/oras-credentials-go/releases/tag/v0.1.1